### PR TITLE
Fixes #2156 make controls visible

### LINKED ISSUE #2156: Keyboard accessibility of control bar after it is hidden during playback
If a user uses tab and enter/space to play the video, focus shifts onto the control bar and the user can continue to use keyboard-only access.

However, once inactivity causes the controls to be hidden, the keyboard cannot be used to restore focus to the controls, so the video player is no longer accessible.

### PR DESCRIPTION
**Pull request by chrisauclair**
## Description
Fixes the a11y issue in #2156, where keyboard and screen reader users lose control focus when the control bar hides due to inactivity. I set the controls to `visibility: visible` and tested with NVDA and JAWS across all supported browsers.

For IE8, I opted to keep the controls hidden because `-ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";` won't make all of the children transparent. More work could be done on the controls for IE8, if someone can make a compelling case for it.

### CODE REVIEW

File: src/css/components/_control-bar.scss [hunk 1]
```diff
@@ -10,6 +10,15 @@
   @include background-color-with-alpha($primary-background-color, $primary-background-transparency);
 }
 
+// IE8 is flakey with fonts, and you have to change the actual content to force
```
Comment (gkatsev) [inline src/css/components/_control-bar.scss]: why move this section?

File: src/css/components/_control-bar.scss [hunk 2]
```diff
@@ -22,11 +31,17 @@
...
 .vjs-has-started.vjs-user-inactive.vjs-playing .vjs-control-bar {
-  visibility: hidden;
+  // Remain visible for screen reader and keyboard users
+  visibility: visible;
   opacity: 0;
 
   $trans: visibility 1.0s, opacity 1.0s;
   @include transition($trans);
+
+  // Make controls hidden in IE8 for now
```
Comment (gkatsev) [inline src/css/components/_control-bar.scss]: sounds good to me

File: src/css/components/_control-bar.scss (hunk 1 above)
Comment (gkatsev) [inline src/css/components/_control-bar.scss]: oh, I see, it's for the variable. Maybe move just the variable but lease the section where it was?

**Comment by gkatsev** [comment]
LGTM.
@OwenEdwards can you take a look? Thanks.

**Comment by OwenEdwards** [comment]
LGTM :+1:

**Pull request closed without merging**